crates/orchestrator/src/lib.rs

```diff
@@ -400,6 +334,137 @@ where
         Ok((node_index, added_to_da))
     }
 
+    fn register_from_list(
+        &mut self,
+        pubkey: &mut Vec<u8>,
+        _da_requested: bool,
```

Comment:
I think we should consider checking if or if not the node should be DA, and only let them connect if it matches their request

Reply:
done!

note that this may require some (mild?) testing, just to make sure that the `is_da` flag is being correctly set in the sequencer. I think it may not have mattered previously